assets/js/util.js

```diff
@@ -120,7 +120,10 @@ $(document).ready(function () {
     resizeFooter();
     $(window)
         .on('hashchange', persistChoices)
-        .resize(resizeFooter);
+        .resize(function () {
+            resizeFooter();
+            populateTranslationList();
```

Comment:
I could do something like:

```
    $(window)
        .on('hashchange', persistChoices)
        .resize(function () {
            resizeFooter();
            if(clock) clearTimeout(clock);
            clock = setTimeout(populateTranslationList, 500);
        });
```

This will ensure that `populateTranslationList` gets fired up only after 500 millseconds pass after window is resized and not immediately. Makes sense?